terraform/test/test.tfvars

```diff
@@ -0,0 +1,2 @@
+model   = "terraform"
+channel = "1.23/edge" # TODO: remove variable once 1.23/stable gets published
```

Comment:
Similar comment to the ui

Reply:
removed TODO in a966a5e77731082712c23b0f5ad6daa52cb57ad8. we can leave `channel = 1.23/edge` in this branch as you suggested